api/api.proto

```diff
@@ -2512,6 +2512,25 @@ service OpenStorageVolume {
           body: "*"
         };
     }
+
+
+}
+
+
+// OpenStorageWatcher is a service used to stream events in the system
+service OpenStorageWatch{
+
+    // Watch streams of event exist in the system
```

Comment:
nit: Watch on resources managed by the driver and receive them as a stream of events.